split_large_bitstream.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+## This script downloads a bitstream from the assetstore, splits it into pieces, and 
+## uploads the pieces to accessible URLs. 
+## Make sure you've set the ASSETSTORE_BUCKET environment variable in your account.
+## Optional arguments are a bitstream ID and the number of chunks to split into.
+
+__author__ = 'daisie'
+
+import re
+import os
+import sys
+import json
+import tempfile
+import subprocess
+from sql_utils import dict_from_query, sql_query
+
+WEB_ASSETS_BUCKET = "dryad-web-assets/temporary"
+ASSETSTORE_BUCKET = os.environ['ASSETSTORE_BUCKET']
+TEMP_DIR = '/transfer-complete/temp'
```

Comment:
This directory won't exist on any machine that wasn't specifically set up for this script. Any reason why we cant use /tmp ?

Reply:
Yes, these files are generally very very large and won't necessarily be able to be stored in /tmp. On the FTP server, /transfer-complete is an extra mount with 1000 GB of space.

Reply:
Ok, then add that note to the script comments and this can be merged.